### ISSUE TITLE
Can't find default.xex even though present

### ISSUE THREAD

**Issue by thomasbra1** [iliazeus/iso2god-rs#5]
Can't find default.xex even though present
Adventure Time - Explore the Dungeon Because I Don'''t Know! (USA, Europe).iso

**Comment by iliazeus** [comment]
Is the `default.xex` really present in the image? It is not in the image file I have access to. (md5 `cf7c9f51c4c71f5b8ba9af3445697068`)

**Comment by iliazeus** [comment]
Sorry, turns out I was looking at system update files that were on this ISO. Will investigate further.

**Comment by iliazeus** [comment]
These are the files `iso2god-rs` currently sees inside this ISO (md5 `cf7c9f51c4c71f5b8ba9af3445697068`). There seems to be no `default.xex`, and no `.xex` files in general.

<details>
<summary>file list</summary>

```
      351 /chunk-042.txt
     1355 /chunk-006_env.txt
        4 /boss-gumwad.ltb
  1022976 /boss-cowcentrifuge.pak
   125632 /blank_gym6.pak
        4 /blank_gym.ltb
  3823488 /autoload.pak
   279104 /adventure_master_duo.pak
  1902432 /abevo.fsb
 11726848 /$SystemUpdate/su20076000_00000000
    30980 /$SystemUpdate/system.manifest
  2598304 /amb.fsb
     1597 /adventure_master_duo.txt
       99 /beemo.txt
  1292992 /beemo.pak
     1107 /autoload.txt
  7292256 /bgvo.fsb
  7496416 /beemovo.fsb
      440 /blank_gym3.txt
   125632 /blank_gym2.pak
      349 /blank_gym.txt
     8640 /blank_gym.pak
   125632 /blank_gym3.pak
      440 /blank_gym2.txt
   125632 /blank_gym5.pak
      440 /blank_gym4.txt
   125632 /blank_gym4.pak
      440 /blank_gym5.txt
        4 /boss-chase.ltb
    59584 /BMO.fsb
      440 /blank_gym7.txt
   125632 /blank_gym7.pak
      440 /blank_gym6.txt
      440 /blank_gym8.txt
   125632 /blank_gym8.pak
  2540992 /boss-chase-props.pak
      108 /boss-cake.txt
   743616 /boss-cake.pak
     3055 /boss-chase-props.txt
   291328 /boss-chase_ns.pak
  1408256 /boss-chase_env.pak
      525 /boss-chase.txt
   346624 /boss-chase.pak
        4 /boss-chase_ns.ltb
     1390 /boss-chase_env.txt
     1344 /boss-chase_ns_env.txt
  1189440 /boss-chase_ns_env.pak
      479 /boss-chase_ns.txt
        4 /boss-cowcentrifuge.ltb
        4 /boss-gbguardians.ltb
   642432 /boss-demoncat_env.pak
      564 /boss-cowcentrifuge_ns.txt
      359 /boss-cowcentrifuge_env.txt
   539968 /boss-cowcentrifuge_env.pak
     1357 /boss-cowcentrifuge.txt
    34368 /boss-cowcentrifuge_ns.pak
        4 /boss-cowcentrifuge_ns.ltb
        4 /boss-demoncat.ltb
      380 /boss-cowcentrifuge_ns_env.txt
   825856 /boss-cowcentrifuge_ns_env.pak
     1220 /boss-demoncat.txt
  1716544 /boss-demoncat.pak
     1137 /boss-demoncat_ns_env.txt
    22528 /boss-demoncat_ns.pak
        4 /boss-demoncat_ns.ltb
```

</details>

**Comment by iliazeus** [comment]
Judging by the file list, it looks like `iso2god-rs` is not able to read the whole list of files - they seem to be ordered alphabetically, and it only gets to letter B.

**Comment by iliazeus** [comment]
The issue was with directory entries spanning multiple sectors in the GDFX filesystem.

Should be fixed in [v1.4.7](https://github.com/iliazeus/iso2god-rs/releases/tag/v1.4.7).
